docs/topics/windows-provisioning-scripts.md

```diff
@@ -0,0 +1,29 @@
+# Windows Provisioning Scripts
+
+## Overview
+
+Due to requirements around signing executable code on Windows nodes some powershell scripts are are delivered via a .zip file during node provisioning.
+This document details how to update and test changes to these files.
+
+## Testing
+
+Changes to the provisioning scripts packaged in the .zip can be tested by:
+
+- Running [scripts/build-windows-provisioning-scripts.sh](../../scripts/build-windows-provisioning-scripts.sh) which will create a .zip and upload it to a storage account
+- Create/update a cluster definition file and set `windowsProfile.provisioningScriptsPackageURL' to point to the uploaded back.
+- Perform a cluster deployment with aks-engine.
+
+## Updates
+
+- Check in script updates to [staging/provisioning/windows](../../staging/provisioning/windows)
+- Create a new **versioned** zip with the contents of [staging/provisioning/windows](../../staging/provisioning/windows) and upload it to a storage account
+  - Optional - First sign the powershell scripts (out-of-scope for this documentation)
+- Update `DefaultWindowsProvisioningScriptsPackageURL` in [pkg/api/const.go](../../pkg/api/const.go) with new URL
+- Add new URL to [/vhd/packer/configure-windows-vhd.ps1::Get-FilesToCacheOnVHD](../../vhd/packer/configure-windows-vhd.ps1)
```

Comment:
Do we ever run this script manually? What is its purpose?

Reply:
Which script are you referring to? 
validate-windowsprovisioning-scripts.sh?
We need to run this as part of a jenkins job or something on the release branches.

Reply:
configure-windows-vhd.ps1

Reply:
That script shouldn't ever be run manually.
It is used by packer in our VHD pipelines.

Reply:
Can we include a section at the top to explain the purpose of this document? To provide context for these instructions and why/when one might use them

Reply:
Added